gadopt/approximations.py

```diff
@@ -411,3 +411,70 @@ def buoyancy(self, p, T):
         pressure_part = self.dbuoyancydp(p, T) * p
         temperature_part = super().buoyancy(p, T)
         return pressure_part + temperature_part
+
+
+class SmallDisplacementViscoelasticApproximation():
+    """Expressions for the small displacement viscoelastic approximation.
+
+    Small displacement linearises the problem. Perturbation about a reference state.
```

Comment:
By assuming a small displacement, we can linearise the problem, assuming a perturbation away from a reference state.